src/stache/data/repo.json

```diff
@@ -0,0 +1 @@
+{"url":"https://github.com/blackbaud/stache2/tree/master/src/app/"}
```

Comment:
This still needed?

Reply:
I still have the updated version of my plugin installed. :P I'll delete this and the plugin and that should stop this from getting added.